Documentation for SinkEnturSales/CLEOS (#8)
TromsFylkestrafikk/ragnarok-entur: SINK-SALES.md

> # Entur Sales

Reviewer: Maybe we could create a folder `src/Docs` in which these files can live, so that we don't clutter the root of the repo?

Author: Agreed, fixed

Sales documentation per agreement.

## Data

> Entur sales provides daily sales data. Including agreement, distribution channel, type of ticket, tickets sold, stop places if indicated, 
> sales time and sum paied. 

Reviewer: ```suggestion
Entur sales provides daily sales data including agreement, distribution channel, type of ticket, tickets sold, sales time, sum paid and stop places if indicated. 
```

Author: Fixed

## Source

> Entur is tne national registry for all public transport in Norway. The registry contains data about daily departures and routes

Reviewer: ```suggestion
Entur is the national registry for all public transport in Norway. The registry contains data about daily departures and routes.
```

Author: Fixed

## Usage

> Get an overview of of number of tickets and types sold on a daily basis. Ticket types can be found in `sales_package_name` 
> like enkeltbillet, enkeltbillett hurtigbåt, 30-dagers billett and more, accompanied by `sales_user_profile_name` like voksen 30-66 år
> Hinnør, Barn and more. Look at schema description for field documentation.

Reviewer: ```suggestion
Get an overview of the amount and type of tickets sold on a daily basis. Ticket types can be found in `sales_package_name` and include among others `enkeltbillett`, `enkeltbillett hurtigbåt` and `30-dagers billett`. `sales_user_profile_name` describes the type of users that purchased the ticket, such as  `voksen 30-66 år`, `honnør` or `barn`. The schema description describes the values in each field.
```

Author: Fixed

> There is one table, `entur_product_sales`. 

Reviewer: ```suggestion
This sink consists of one table called `entur_product_sales`. 
```

Author: Fixed
